src/scripts/main.js

```diff
@@ -1 +1,97 @@
+/* eslint-disable no-console */
+/* eslint-disable no-shadow */
 'use strict';
+
+function showNotification(message, isError = false) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.className = isError ? 'error' : 'success';
+  div.textContent = message;
+  document.body.appendChild(div);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  let clicked = false;
+
+  function onLeftClick(event) {
+    if (event.button === 0) {
+      clicked = true;
+      resolve('First promise was resolved on a left click');
+      document.removeEventListener('mousedown', onLeftClick);
+    }
+  }
+
+  document.addEventListener('mousedown', onLeftClick);
+
+  setTimeout(() => {
+    if (!clicked) {
+      reject(
+        new Error('First promise was rejected in 3 seconds if not clicked'),
+        document.removeEventListener('mousedown', onLeftClick),
+      );
```

Comment:
The `reject` function should only take a single argument. Currently, you are passing multiple arguments to `reject`, which is incorrect. You should remove the `document.removeEventListener('mousedown', onLeftClick)` from the `reject` call and place it before or after the `reject` call.